hack/make-rules/test.sh

```diff
@@ -300,7 +307,6 @@ runTests() {
     produceJUnitXMLReport "${junit_filename_prefix}"
     return ${rc}
   fi
-
```

Comment:
nit: Maybe we don't need to remove this blank line because blocks which starts with comment have usually a blank line just before that.

Reply:
Thanks for review and comment. I removed it unintentionally. I reverted the blank line.

Reply:
@oomichi I rebased my patch because vet.sh in master branch was changed. Could you review again?